zubhub_backend/zubhub/projects/migrations/0007_tag_tag_name_gin_idx.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2 on 2022-11-07 04:01
```

Comment:
We remove this migration in the past because of the error it was causing in production. This error was due to the changes made to table index by Andrew (it is somehow conflicting with the name of an already existing index). Here we had to add it back because we can no longer avoid it. while we can continue iterating on this feature in our local dev environments, we can't merge this PR to master without fixing the bug that this migration file introduces and attempting to do so WILL CAUSE AN ERROR IN PRODUCTION.

Reply:
This is now resolved after @NdibeRaymond 's help in production! :pray: